serverless.yml

```diff
@@ -16,6 +16,8 @@ provider:
     LICENSE_KEY: ${env:LICENSE_KEY}
     # Determines if logs are forwarded to New Relic Logging
     LOGGING_ENABLED: ${env:LOGGING_ENABLED, "False"}
+    # A boolean to determine if you want to output debug messages in the CloudWatch console
+    DEBUG_LOGGING_ENABLED: ${env:DEBUG_LOGGING_ENABLED}
```

Comment:
This should have a default value. See lines above for the syntax for that.

Reply:
Good catch, thanks!